src/main/java/taxi/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package taxi.service;
+
+import java.util.Optional;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverService driverService;
+
+    @Override
+    public Driver login(String login, String password) throws AuthenticationException {
+        Optional<Driver> driver = driverService.findByLogin(login);
```

Comment:
Consider using `orElseThrow` instead of `if (driver.isEmpty())` to simplify the code and throw an exception when the driver is not found.